plotly/plotly/plotly.py

```diff
@@ -1449,6 +1454,195 @@ def _send_to_plotly(figure, **plot_options):
+        place the plot in it.
+    :param (str) sharing: see `plotly.plotly.plot()` doc string.
+    :param (bool) auto_open: if True, opens plot in the browser. If False,
+        returns the url for the plot instead.
+
+    Example 1: Simple Animation
+    ```
+    import plotly.plotly as py
+    from plotly.grid_objs import Grid, Column
+
+    column_1 = Column([1, 2, 3], 'x')
+    column_2 = Column([1, 3, 6], 'y')
+    column_3 = Column([2, 4, 6], 'new x')
+    column_4 = Column([1, 1, 5], 'new y')
+    grid = Grid([column_1, column_2, column_3, column_4])
+    py.grid_ops.upload(grid, 'animations_grid', auto_open=False)
+
+    # create figure
+    figure = {
+        'data': [
+            {
+                'xsrc': grid.get_column_reference('x'),
+                'ysrc': grid.get_column_reference('y')
+            }
+        ],
+        'layout': {'title': 'First Title'},
+        'frames': [
+            {
+                'data': [
+                    {
+                        'xsrc': grid.get_column_reference('new x'),
+                        'ysrc': grid.get_column_reference('new y')
+                    }
+                ],
+                'layout': {'title': 'Second Title'}
+            }
+        ]
+    }
+
+    py.create_animations(figure, 'new_plot_with_animations')
+    ```
+    """
+    credentials = get_credentials()
+    validate_credentials(credentials)
+    username, api_key = credentials['username'], credentials['api_key']
+    auth = HTTPBasicAuth(str(username), str(api_key))
+    headers = {'Plotly-Client-Platform': 'python',
+               'content-type': 'application/json'}
+
+    json = {
+        'figure': figure,
+        'world_readable': True
+    }
+
+    # set filename if specified
+    if filename:
+        # warn user that creating folders isn't support in this version
+        if '/' in filename:
+            warnings.warn(
+                "This BETA version of 'create_animations' does not support "
+                "automatic folder creation. This means a filename of the form "
+                "'name1/name2' will just create the plot with that name only."
+            )
+        json['filename'] = filename
+
+    # set sharing
+    if sharing == 'public':
+        json['world_readable'] = True
+    elif sharing == 'private':
+        json['world_readable'] = False
+    elif sharing == 'secret':
+        json['world_readable'] = False
+        json['share_key_enabled'] = True
+    else:
+        raise exceptions.PlotlyError(
+            "Whoops, sharing can only be set to either 'public', 'private', "
+            "or 'secret'."
+        )
+
+    api_url = _api_v2.api_url('plots')
+    r = requests.post(api_url, auth=auth, headers=headers, json=json)
+    r.raise_for_status()
+
+    try:
+        parsed_response = r.json()
+    except:
+        parsed_response = r.content
+
+    if 'error' in r and r['error'] != '':
+        raise exceptions.PlotlyError(r['message'])
```

Comment:
huh? Wouldn't this want to print out `r['error']` If that's not the case can you add a quick code comment? This is a little confusing to me.